### ISSUE TITLE
Компоненты

### ISSUE THREAD

**Issue by glutaminefree** [kamirt/simple-calc#1]
https://github.com/kamirt/simple-calc/blob/c41e214ff27f7f63dc08dc78c57ed3190315cf29/src/components/Plus.vue
https://github.com/kamirt/simple-calc/blob/c41e214ff27f7f63dc08dc78c57ed3190315cf29/src/components/Equals.vue

Серьёзно? Компонент чтобы отобразить знак равенства и знак плюса?
<сарказм>Нужно было сделать отдельный компонент на каждую полоску в знаке равно, а для плюса один компонент с вертикальной линией и другой с горизонтальной. Хотя плюс можно было сделать ещё круче - один компонент - линия вверх, ещё один компонент - линия вправо, третий компонент с линией вниз и последний компонент с линией влево! Тогда точно было бы огонь!</сарказм>

**Comment by kamirt** [comment]
@glutaminefree,  ну мой сарказм первее был) Обычно работодатели как раз очень придираются к модульности - вот вам модульность. Я постарался таким образом показать, что могу, умею, практикую разбивать на компоненты, пробрасывать события на родителя через два компонента. В рамках такого небольшого приложения, мне показалось, что от меня именно такого подхода и ждали. Ну или по крайней мере, оно лишним не будет в этом случае. Понятное дело, что в нормальном приложении количество папок и файлов просто зашкалит при таком подходе и поверьте, у меня присутствет чувство меры =)

**Comment by glutaminefree** [comment]
@kamirt Ну это меняет дело. Но я не вижу никаких пробросов событий или какой-то обработки данных внутри этих компонентов. Только вёрстка. Нам нужно поддерживать и развивать серьёзный проект, а не шутки шутить. Хотя шутки я тоже очень люблю ;) Но они должны быть к месту и подходящими.

**Comment by kamirt** [comment]
@glutaminefree приглядитесь. Из компонента, где инпут событие файрится в самый верхний. Какой, как говорится, вопрос, такой и ответ. Это во-первых. Во-вторых, вы кто? Вы дали мне это задание? Не считаете нужным представиться? Если так, вы получили его полностью рабочим, код довольно чист, никаких претензий по существу я не вижу. В чем смысл сарказма, если вам некогда шутки шутить? Как правило, так ведут себя либо школьники, либо мужчины с весьма коротким половым органом или импотенцией, пытаясь самоутвердиться за чей-то счет.  Во всех этих случаях, мы не сработаемся.

**Comment by kamirt** [comment]
@glutaminefree собственно, да. У тех, у кого "серьезный проект" меньше всего ожидаешь увидеть первым репозиторием плагин для вордпресс. Удачи.

**Comment by glutaminefree** [comment]
@kamirt 

> У тех, у кого "серьезный проект" меньше всего ожидаешь увидеть первым репозиторием плагин для вордпресс

Ага, последний коммит в который был сделан 4 года назад ;) Вы очередной вредный капризулька, который считает себя всезнающим? Извините тогда, что заняли ваше драгоценное время.

**Comment by kamirt** [comment]
@glutaminefree нет, не извиню. Я с удовольствием приставил бы к вам пистолет и заставил бы тоже делать по нескольку вот этих тестовых в день, бесполезных и бесплатных, а потом исходил бы сарказмом и называл "капризулькой". Потом посмотрел бы, как вам понравится профессия. У русских работодателей и заказчиков, убеждаюсь в который раз, нет ничего, кроме наглости, морального уродства, болезненной спеси и жадности. Как и у политиков, но это вне темы. Потом жалуйтесь дальше, что специалисты все бегут из страны. Рано или поздно, только вы тут и останетесь. Удачи.

**Comment by glutaminefree** [comment]
@kamirt Вам не тяжело столько негатива в голове своей носить?

>заставил бы тоже делать по нескольку вот этих тестовых в день, бесполезных и бесплатных

Это особенности профессии. Можно принять это как данность, если есть желание работать в данной сфере.

>У русских работодателей и заказчиков, убеждаюсь в который раз, нет ничего, кроме наглости, морального уродства, болезненной спеси и жадности.

У нас всё иначе. Просто нужно делать свою работу с профессионализмом и пониманием, а не раздувать своё эго до размеров Солнечной системы и делать "шуточные" тестовые задания, которые никто не оценит. А что вы вообще ожидали, кстати, сделав такую работу, в которой вынесены в компоненты просто единичные символы? Что вас за это позовут в топовые IT конторы?
Если вы на этапе тестового задания выказываете свою неоправданную спесь и показываете не оптимизированный код ради шутки, то не спешите сыпать оскорблениями в сторону работодателя. Вас так вряд ли кто-то позовёт работать к себе.

Я вам желаю всех благ и радостной жизненной обстановки.
